lib/facter/self_service.rb

```diff
@@ -112,4 +112,17 @@
     # Is there at least 9% memory available
     { S0021: Facter.value(:memory)['system']['capacity'].to_f <= 90 }
   end
+  
+  chunk(:S0033) do
```

Comment:
```suggestion
  chunk(:S0033) do
  next unless PuppetSelfService.replica? || PuppetSelfService.compiler? || PuppetSelfService.legacy_compiler? || PuppetSelfService.primary
```

This needs confined to only those things with a puppetserver

Reply:
Added this!